src/channels/media.rs

```diff
@@ -364,6 +504,56 @@ where
     {
         let request_id = self.message_manager.generate_request_id();
 
+        let metadata = media.metadata.as_ref().map(|m| match *m {
+            Metadata::Generic(ref x) => proxies::media::Metadata {
+                title: x.title.clone(),
+                subtitle: x.subtitle.clone(),
+                images: x.images.iter().map(|i| i.encode()).collect(),
+                release_date: x.release_date.clone(),
+                .. proxies::media::Metadata::new(0)
+            },
+            Metadata::Movie(ref x) => proxies::media::Metadata {
+                title: x.title.clone(),
+                subtitle: x.subtitle.clone(),
+                studio: x.studio.clone(),
+                images: x.images.iter().map(|i| i.encode()).collect(),
+                release_date: x.release_date.clone(),
+                .. proxies::media::Metadata::new(1)
+            },
+            Metadata::TvShow(ref x) => proxies::media::Metadata {
+                series_title: x.series_title.clone(),
+                subtitle: x.episode_title.clone(),
+                season: x.season,
+                episode: x.episode,
+                images: x.images.iter().map(|i| i.encode()).collect(),
+                original_air_date: x.original_air_date.clone(),
+                .. proxies::media::Metadata::new(2)
+            },
+            Metadata::MusicTrack(ref x) => proxies::media::Metadata {
+                album_name: x.album_name.clone(),
+                title: x.title.clone(),
+                album_artist: x.album_artist.clone(),
+                artist: x.artist.clone(),
+                composer: x.composer.clone(),
+                track_number: x.track_number,
+                disc_number: x.disc_number,
+                images: x.images.iter().map(|i| i.encode()).collect(),
+                release_date: x.release_date.clone(),
+                .. proxies::media::Metadata::new(3)
+            },
+            Metadata::Photo(ref x) => proxies::media::Metadata {
+                title: x.title.clone(),
+                artist: x.artist.clone(),
+                location: x.location.clone(),
+                latitude: x.latitude_longitude.map(|coord| coord.0),
+                longitude: x.latitude_longitude.map(|coord| coord.0),
+                width: x.dimensions.map(|dims| dims.0),
+                height: x.dimensions.map(|dims| dims.0),
```

Comment:
__issue:__ same here, `dims.1` for `height`?

Reply:
Good catch, fixed.